src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,6 +3,7 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final String exceptionText = "Index out of bound exception.";
```

Comment:
constants write in uppercase, words divided by "_"

Reply:
+